### ISSUE TITLE
cannot enter more than 1 number to send emergency message to

### ISSUE THREAD

**Issue by bxhghz** [SaferMobile/InTheClear#38]
I am using a Nokia 2700; I am trying to enter 2 numbers in the Emergency message, but I cannot draw a comma/cannot enter another number

**Comment by djhalliday** [comment]
agreed to limit to a single recipient for the current release. means the copy must change to reflect this.

**Comment by n8fr8** [comment]
"Now you must setup the actual emergency mobile phone number to use and message to send in case of an emergency."

is now set.
